OpenUpSA/wazimap-ng-ui
# removing no data message from category descriptions & displaying descriptions correctly

## Description
this PR will replace https://github.com/OpenUpSA/wazimap-ng-ui/pull/427

We should not indicate "no data available" outside of an indicator

![image](https://user-images.githubusercontent.com/53019884/132109060-365fc745-b1d6-42c4-b004-b8f9c29eabc8.png)


## Related Issue
https://wazimap.atlassian.net/browse/WNCM-62

## How to test it locally
* go to SANEF profile
* expand rich data panel
* confirm that we don't show "No data available for this indicator for this geographic area" message outside indicators - especially in category/subcategory descriptions


## Screenshots


## Changelog

### Added

### Updated
* `profile/subcategory.js` to remove the unnecessary error message
* `profile/category.js` & `profile/subcategory.js` to remove `.hidden` from description element when there is description to display
* `__tests__/profile/rich_data.test.js` to add a unit test

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
